bin/uat_deploy

```diff
@@ -37,7 +42,8 @@ deploy() {
                 --set image.repository="$IMG_REPO" \
                 --set image.tag="${CIRCLE_SHA1}" \
                 --set ingress.hosts="{$RELEASE_HOST}" \
-                --set replicaCount="1"
+                --set replicaCount="1" \
+                --timeout 60
```

Comment:
the default one is 300, isn't it? Did you mean to reduce it from 5 minutes to 1 minute?